pavelib/paver_tests/test_xsscommitlint.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
Thanks @andy-armstrong. The history did get lost.  Keeping the renames to a separate commit seems to now keep the history locally, but GitHub is still having issues on these two files.  Not sure what is different about that from the other files, but I think we are good with history now.